changelogs/fragments/1064_kubevirt_tests.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- kubevirt - re-enable tests after refactor of KubernetesRawModule (https://github.com/ansible-collections/community.general/issues/1064).
```

Comment:
Test-only PRs do not need a changelog fragment. (There also wasn't one for disabling the tests.)